### ISSUE TITLE
Incognito Block Timer ticks down on Non-Private tabs

### ISSUE THREAD

**Issue by leoa9001** [proginosko/LeechBlockNG-chrome#83]
I have a block-set that is set for "only-private-tabs" and blocks pages "within time periods and after time limit." I noticed that the countdown timer for it ticks down when sites in the list are open on non-private tabs. Is this the intended functionality and is there a way to make a timer that only counts down when sites are open in private tabs?

**Comment by proginosko** [comment]
Good catch! You're right; that's a bug (and a pretty embarrassing one too). I will get that fixed in the next release.

**Comment by proginosko** [comment]
Fixed in version 1.6.8.
